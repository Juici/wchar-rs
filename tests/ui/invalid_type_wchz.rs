use wchar::wchz;

const INVALID_TYPE: &[f32] = wchz!(f32, "oops");

fn main() {}
