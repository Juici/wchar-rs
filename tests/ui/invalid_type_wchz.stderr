error: expected one of: `u16`, `u32`, `i16`, `i32`
 --> $DIR/invalid_type_wchz.rs:3:30
  |
3 | const INVALID_TYPE: &[f32] = wchz!(f32, "oops");
  |                              ^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `wchz` (in Nightly builds, run with -Z macro-backtrace for more info)
